tests/benchdnn/graph/flex_rewrite.cpp

```diff
@@ -1393,51 +1412,51 @@ void flex_rewrite_t::dt_rewrite(deserialized_graph_t &dgraph) {
 }
 
 void flex_rewrite_t::op_kind_rewrite(deserialized_graph_t &dgraph) {
-    // Step 1: check the op kind in the map and whether the given ids are in
-    // the graph.
-    for (const auto &v : op_kind_map_) {
-        if (v.second == "default") return;
+    if (op_kind_map_.size() == 1 && op_kind_map_.begin()->second == "default")
+        return;
 
-        auto target_kind = opstr2kind(v.second);
-        if (target_kind == dnnl::graph::op::kind::LastSymbol) {
-            BENCHDNN_PRINT(0,
-                    "graph: rewrite: invalid target op kind %s is provided\n",
-                    v.second.c_str());
-            SAFE_V(FAIL);
-        }
+    for (auto &aop : dgraph.ops_) {
+        for (const auto &v : op_kind_map_) {
```

Comment:
Nit: less unneeded spaces this way (no curls on the first and last lines)
```suggestion
    for_(auto &aop : dgraph.ops_)
    for (const auto &v : op_kind_map_) {
```